src/components/10-atoms/toggle-switch/index.scss

```diff
@@ -0,0 +1,60 @@
+$toggle-switch-animation: 0.4s ease;
+$toggle-switch-height: 30px;
+$toggle-switch-width: 50px;
+$toggle-switch-spacing: 2px;
+
+.a-toggle-switch {
+  display: block;
+  position: relative;
+  height: $toggle-switch-height;
+  width: $toggle-switch-width;
+
+  &__input {
+    display: none;
```

Comment:
Please ask @markus-walther why he dont just set `display: none` at the `checkbox`component. His code there is: 
```
.a-checkbox__input {
    position: absolute;
    width: 1px;
    height: 1px;
    clip: rect(0, 0, 0, 0);
    margin: -1px;
    overflow: hidden;
  } 
```